hello-strongbox-gradle/build.gradle

```diff
@@ -9,8 +13,35 @@ compileJava {
 
 description = "Example project that shows how to use gradle to build and deploy artifacts to Strongbox"
 
-ext.username = System.getProperty('credentials.username') != null ? System.getProperty('credentials.username') : mavenUser
-ext.password = System.getProperty('credentials.password') != null ? System.getProperty('credentials.password') : mavenPassword
+// publishing to GitHub Package Registry
+publishing {
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/j4ckofalltrades/strongbox-examples")
+            credentials {
+                username = project.findProperty("gpr.user") ?: System.getenv("GITHUB_USERNAME")
+                password = project.findProperty("gpr.key") ?: System.getenv("GITHUB_TOKEN")
```

Comment:
Do we need to have Github secrets defined for these?

Reply:
No, `GITHUB_USERNAME` is sourced from `actions/setup-java` while `GITHUB_TOKEN` is automatically set by GitHub for each repository.

I'll document this here.

Reply:
Excellent, thank you very much! :)

It might be worth mentioning all this in the master task for future reference.

Reply:
Yeah that's a good idea. I'll drop in a comment there.